Sources/Hummingbird/Middleware/LogRequestMiddleware.swift

```diff
@@ -72,28 +72,28 @@ public struct LogRequestsMiddleware<Context: RequestContext>: RouterMiddleware {
                 level: self.logLevel,
                 "Request",
                 metadata: [
-                    "hb_uri": .stringConvertible(request.uri),
-                    "hb_method": .string(request.method.rawValue),
+                    "hb.uri": .stringConvertible(request.uri),
+                    "hb.method": .string(request.method.rawValue),
                 ]
             )
         case .all(let except):
             context.logger.log(
                 level: self.logLevel,
                 "Request",
                 metadata: [
-                    "hb_uri": .stringConvertible(request.uri),
-                    "hb_method": .string(request.method.rawValue),
-                    "hb_headers": .stringConvertible(self.allHeaders(headers: request.headers, except: except)),
+                    "hb.uri": .stringConvertible(request.uri),
+                    "hb.method": .string(request.method.rawValue),
+                    "hb.headers": .stringConvertible(self.allHeaders(headers: request.headers, except: except)),
```

Comment:
```suggestion
                    "http.request.path": .stringConvertible(request.uri),
                    "http.request.method": .string(request.method.rawValue),
                    "http.request.headers": .stringConvertible(self.allHeaders(headers: request.headers, except: except)),
```

I believe these libs also support providing an array of values right? We should consider emitting headers as an array or dict.

Reply:
`http.request.path` and `http.request.headers` are not official spec. So I'm not 100% on this

Reply:
For non-spec stuff maybe we should just keep all of these under the `hb` namespace